automatic/qtox/legal/VERIFICATION.txt

```diff
@@ -1,21 +1,21 @@
-VERIFICATION
+﻿VERIFICATION
 Verification is intended to assist the Chocolatey moderators and community
 in verifying that this package's contents are trustworthy.
 
 The embedded software have been downloaded from the listed download
-location on <https://github.com/qTox/qTox/blob/master/README.md>
+location on <https://qtox-win.pkg.tox.chat/qtox/win64>
```

Comment:
Since 32-Bit and 64-Bit version are embedded, https://qtox-win.pkg.tox.chat/qtox/ would be a better fit